src/sdk/models/policyStatistics.ts

```diff
@@ -0,0 +1,36 @@
+import { IEntityIdentifier } from './base';
+
+export interface PolicyStatistic {
+	name?: string;
+	description?: string;
+	bytes?: number;
+	files?: number;
+	folders?: number;
+	versions?: number;
+	storage?: number;
+	cost_savings?: number;
+}
+
+export interface PolicyStatisticsList {
+	[key: string]: PolicyStatistic;
+}
+
+export type PolicyJobStatisticsTimeline = { 
+	timestamp: number; 
+	stats: {
+		by_risk?: PolicyStatisticsList;
+		by_group?: PolicyStatisticsList;
+		by_activity?: PolicyStatisticsList;
```

Comment:
`by_activity` will not be populated for phase 1. We will need to handle that correctly from the UI.